### ISSUE TITLE
.SaveAs(file) from template gives: NullReferenceException: Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by charneca** [EPPlusSoftware/EPPlus#289]
On latest Prerelease  5.5.2.4 I can't save excel file after making changes to a specific template with some pivot tables and pivot charts.

on version 5.0.4 it saves correctly.

file:
[test1.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5881945/test1.xlsx)

test code:  
```
        public class ReportTest
        {
            public string ConfigurationItem { get; set; }
            public int Count { get; set; }
        }

        public void Test1()
        {

            string workingDirectory = AppDomain.CurrentDomain.BaseDirectory;
            FileInfo template = new(workingDirectory + @"Templates\test1.xlsx");
            FileInfo finalFile = new(string.Format(workingDirectory + @"Cache\test1_{0}.xlsx", DateTime.Now.Ticks));

            finalFile.Delete();


            using (var excelFile = new ExcelPackage(template))
            {

                var topCI = new List<ReportTest>
                {
                    new ReportTest{ ConfigurationItem = "Item1", Count =3 },
                    new ReportTest{ ConfigurationItem = "Item2", Count =3 },
                    new ReportTest{ ConfigurationItem = "Item3", Count =3 }
                };

                ////Top CI
                var worksheet = excelFile.Workbook.Worksheets["Top CI"];
                var location = worksheet.Cells["Z1"].LoadFromCollection(Collection: topCI, PrintHeaders: true);
                worksheet.Tables.Add(location, "TopCiTypeTbl" + DateTime.Now.Ticks);


                //change month and select first worksheet
                var wsDashBoards = excelFile.Workbook.Worksheets["Dashboards"];

                wsDashBoards.Cells["A1"].Value = "Test Report";

                wsDashBoards.Select();


                var test = worksheet.PivotTables;

                excelFile.SaveAs(finalFile);
            }
        }
```

What I do to create the templates is: I first create the template with pivot tables/charts and data then remove them, and save the file as template. 
Epplus then add the new pivot tables with data...

**Comment by JanKallman** [comment]
Fixed in latest build.

**Comment by JanKallman** [comment]
Occured when pivot tables referenced empty ranges without a header value on the first row of the columns.

**Comment by JanKallman** [comment]
Fixed in 5.5.3

**Comment by charneca** [comment]
> Fixed in 5.5.3

Hello,

Save now works, but the excel file gets corrupted:

![error1](https://user-images.githubusercontent.com/61232803/107239248-91a5ec00-6a20-11eb-957c-9a42087b0a70.jpg)

![error](https://user-images.githubusercontent.com/61232803/107238732-16443a80-6a20-11eb-8d1e-4cba8034812c.jpg)

On version 5.0.4 the file seems to be saved correctly.
